packages/connect/CHANGELOG.md

```diff
@@ -1,15 +1,54 @@
-|             Package              | Stable | Canary |
-| :------------------------------: | :----: | :----: |
-|       npm @trezor/connect        | 9.2.4  |   -    |
-|     npm @trezor/connect-web      | 9.2.4  |   -    |
-| npm @trezor/connect-webextension | 9.2.4  |   -    |
+|             Package              | Stable |    Canary    |
+| :------------------------------: | :----: | :----------: |
+|       npm @trezor/connect        | 9.2.4  | 9.2.5-beta.1 |
+|     npm @trezor/connect-web      | 9.2.4  | 9.2.5-beta.1 |
+| npm @trezor/connect-webextension | 9.2.4  | 9.2.5-beta.1 |
 
-|     Deployment      | Stable | Canary |
-| :-----------------: | :----: | :----: |
-| connect.trezor.io/9 | 9.2.4  |   -    |
+|     Deployment      | Stable |    Canary    |
+| :-----------------: | :----: | :----------: |
+| connect.trezor.io/9 | 9.2.4  | 9.2.5-beta.1 |
 
 User the persistent link connect.trezor.io/9 to access the latest stable version of Connect Explorer.
 
+# 9.2.5-beta.1
```

Comment:
@karliatto @martykan 
Please try to write some broader copy so that the commit messages are translated to human language. 😄 I can help, but you have to start.

Reply:
good enough I'd say  👍

Reply:
That reads quite human friendly. :+1: